### PR TITLE
Added Anubhav Adhikari's introduction

### PR DESCRIPTION
**Pull request by anubhav1206**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,4 @@
...
 - Hello, I am Pushkaraj Kulkarni, a professional Full-Stack Developer and an open-source contributor. I specialize in web development and am currently available for freelance work. My skill set includes expertise in programming languages such as JavaScript, Typescript, and Python. I'm specialized in frameworks like React, Next.js, and Express.js - [Pushkaraj Kulkarni](https://github.com/pushkaraj2007)
+- Good Evening my name is Anubhav and I am an aspiring developer and a problem solver with expertise in various technologies. I have contributed to numerous projects and have gained recognition for my work. I consider myself fairly creative and a hardworking individual. I have a passion for technology and gaming. I have hands-on experience in building real-world projects and keep myself updated with all the news related to gaming and tech. When I’m not coding, you can find me playing games and exploring new technologies.
```
Comment (Panquesito7) [inline README.md]:
```suggestion
- Good Evening my name is Anubhav and I am an aspiring developer and a problem solver with expertise in various technologies. I have contributed to numerous projects and have gained recognition for my work. I consider myself fairly creative and a hardworking individual. I have a passion for technology and gaming. I have hands-on experience in building real-world projects and keep myself updated with all the news related to gaming and tech. When I’m not coding, you can find me playing games and exploring new technologies. - [Anubhav Adhikari](https://github.com/anubhav1206)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (KendallDoesCoding) [reply README.md]: @anubhav1206 - please make these changes to proceed.

**Comment by KendallDoesCoding** [comment]
@The-Lively-Developers-Community/maintainers - can someone help me resolve merge conflicts?

**Review by Panquesito7 — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
@anubhav1206 - thanks for adding your introduction. Make sure to join our community, by creating an issue here: https://github.com/The-Lively-Developers-Community/become-a-member :)!

**Pull request merged**